docs/variables/user.md

```diff
@@ -4,137 +4,410 @@ id: user
 
 # $(user)
 
-This variable returns either the first argument in the command, or if that is unavailable, the username of the person who triggered the command, or a custom specified user (for example `$(user aiden)` would use `aiden` as the user).
+Returns a range of metadata around a (specified) user's channel that can be accessed via multiple different variable members for each unique piece of information.
```

Comment:
apos.

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/18#discussion_r786576213).